### PR TITLE
test: merge two similar test user session utils

### PR DESCRIPTION
**Pull request by hamidzr**
## Description

` determined_test_sessions`

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

File: e2e_tests/tests/api_utils.py [hunk 1]
```diff
@@ -7,7 +7,9 @@
 
 def determined_test_session(
     credentials: Optional[authentication.Credentials] = None,
+    admin: Optional[bool] = None,
```
Comment (hamidzr) [inline e2e_tests/tests/api_utils.py:10]: wait this actually needs to do something. brb

**Comment by hamidzr** [comment]
should be good for a quick review after the tests pass cc @ioga @nrajanee

File: e2e_tests/tests/experiment/test_core.py [hunk 2]
```diff
@@ -8,6 +8,7 @@
 from determined.common import yaml
 from determined.common.api import bindings
 from determined.experimental import Determined
+from tests import api_utils
```
Comment (hamidzr) [inline e2e_tests/tests/experiment/test_core.py:11]: I think mypy or isort used merge these imports into one line

**Comment by hamidzr** [comment]
happy to either land these through this pr or close and wait for https://github.com/determined-ai/determined/pull/5548

**Review by nrajanee — APPROVED** [review]

**Pull request merged**